_rules/form-field-non-empty-accessible-name-e086e5.md

```diff
@@ -45,7 +45,7 @@ _There are currently no assumptions_
 
 The list of roles in the applicability is derived by taking all the roles from [WAI-ARIA Specifications](#wai-aria-specifications) that:
 
-- have [semantic roles][] that inherit from the [abstract](https://www.w3.org/TR/wai-aria/#abstract_roles) `input` or `select` role, and
+- have [semantic roles][] that inherit from the `input`, `menuitem` or `select` role, and
```

Comment:
Can you also remove line 42 (acc support on option). That line is in the wrong section.

```suggestion
- have [semantic roles][] that inherit from the `input`, `menuitem` or `select` role, and
- are not `menu` or `option`, since these are not form controls, and
```